app/src/main/java/cm/aptoide/pt/install/installer/Root.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * Aurora Store
+ * Copyright (C) 2019, Rahul Kumar Patel <whyorean@gmail.com>
+ *
+ * Aurora Store is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * Aurora Store is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Aurora Store.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ */
+
+package cm.aptoide.pt.install.installer;
+
+import android.util.Log;
+import cm.aptoide.pt.logger.Logger;
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStreamWriter;
+
```

Comment:
Please add the direct link to the gitlab repo class so people know where was it taken from and where to look for updates.